README.md

```diff
@@ -2,6 +2,10 @@
 
 This page will give you an overview of HOD DSP, the services it entails and the ways to get access to them. This is a general overview. Developers should check out the [Developer Docs](https://github.com/UKHomeOffice/hosting-platform/tree/master/developer-docs).
 
+## Service Status
+
+For an overview of the platform services we provide and their current status i.e. Alpha, Beta, Live, please refer to the [platform service status page](docs/platform_service_status.md). This is to give consumers of the platform a single reference point on the services we are providing and where each service is in the lifecycle. For definitions on each phase of the lifecycle and the criteria for production-ready; please refer [here](docs/service_lifecycle.md).
+
```

Comment:
I meant the second semi colon, using a semi colon after "Alpha, Beta, Live;" is totally correct